### PR TITLE
Update to allow json input for locations

### PR DESCRIPTION
**Pull request by dgearhart**

### CODE REVIEW

**Comment by gknisely** [comment]
:ship:

File: src/thor/pathtest/pathtest.cc [hunk 1]
```diff
@@ -225,15 +228,63 @@ int main(int argc, char *argv[]) {
...
+        std::cerr << "The <" << arg
+                  << "> argument was not provided, but is mandatory when json is not provided\n\n";
+        std::cerr << options << "\n";
+        return EXIT_FAILURE;
+      }
+    }
+    originloc = Location::FromCsv(origin);
+    destloc = Location::FromCsv(destination);
+  } else {
+    std::cout << "json=" << json << std::endl;
```
Comment (kevinkreiser) [inline src/thor/pathtest/pathtest.cc]: dont need this line i guess

**Comment by dnesbitt61** [comment]
:ship:

File: src/thor/pathtest/pathtest.cc (hunk 1 above)
Comment (dgearhart) [inline src/thor/pathtest/pathtest.cc]: yep - thanks

**Pull request merged**